### PR TITLE
[WIP] Plumb the log level through to plugin processes

### PR DESCRIPTION
**Pull request by ncdc**
Signed-off-by: Andy Goldstein <andy.goldstein@gmail.com>

### CODE REVIEW

**Comment by ncdc** [comment]
This only does it currently for the Ark internal plugins. I think we should do it for external plugins too.

File: pkg/plugin/logger.go [hunk 1]
```diff
@@ -24,8 +24,9 @@ import (
 
 // NewLogger returns a logger that is suitable for use within an
 // Ark plugin.
-func NewLogger() logrus.FieldLogger {
+func NewLogger(level logrus.Level) logrus.FieldLogger {
```
Comment (ncdc) [inline pkg/plugin/logger.go]: This is a breaking change to plugins and will require an update in the ark-plugin-example repo

**Comment by ncdc** [comment]
We potentially could update `arkplugin.NewServer` to make it support `--log-level`

**Comment by nrb** [comment]
@disrani-px @edevenport This change will be coming in v0.10.1 and will require another update to plugins once we merge it.

**Comment by king-jam** [comment]
This going to be merged in v0.11.1 or v0.12.1?

**Comment by skriss** [comment]
@king-jam probably `v0.12.0`

**Pull request closed without merging**